src/lucky_cli/task.cr

```diff
@@ -7,9 +7,9 @@ abstract class LuckyCli::Task
     end
   end
 
-  macro banner(banner_text)
-    def banner
-      {{banner_text}}
+  macro summary(summary_text)
+    def summary
+      {{summary_text}}
     end
   end
 
```

Comment:
Could you add another macro for `banner` that just raises and says to use `summary` instead? Something like this:

```crystal
macro banner
  {% raise "'banner' has been renamed to 'summary'" %}
end
```